x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/JdbcAssert.java

```diff
@@ -34,6 +38,15 @@
  * Utility class for doing JUnit-style asserts over JDBC.
  */
 public class JdbcAssert {
+
+    private static final IntObjectHashMap<DataType> SQL_TO_TYPE = new IntObjectHashMap<>();
```

Comment:
Because the QA project doesn't see JDBC driver. And in the future the `TypeUtils` will be limited to the jdbc package alone. As an upside this one uses a different map implementation.